packages/patternfly-4/react-core/src/components/Progress/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './Progress';
+export * from './ProgressContainer';
```

Comment:
Change to `export { ProgressMeasureLocation, ProgressVariant } from './ProgressContainer';`